### DISCUSSION TITLE
Question: will I lose map switching from Roborock to Xiaomi Home

### DISCUSSION THREAD

**Discussion by weitheng** [PiotrMachowski/Xiaomi-cloud-tokens-extractor#33] [Q&A]
I am currently using the Roborock app. Have been wanting to get the vacuum robot integrated to Home Assistant, but have been putting that off as I am unsure whether the maps that have been mapped out will be gone when switching to the Xiaomi Home app to extract the tokens.

Appreciate it if you could give me you kind advice. Thanks

**Comment by PiotrMachowski** [comment]
As far as I know you have to reset the vacuum to migrate it to Xiaomi Home, so the maps will probably be gone

**Comment by ryanelliottsmith** [comment]
I just switched my S6 from the Roborock app to the Mi Home app(iOS) and didn't lose the map. You only reset wifi, not the whole device.

**Reply by weitheng** [reply]
That’s good to hear! Did you have multi-level mapping previously, is it still accessible on the Mijia app?
